### ISSUE TITLE
Remove internal presets

### ISSUE THREAD

**Issue by markwmuller** [mailmindr/issues#2]
**Is your feature request related to a problem? Please describe.**
There are too many presets for follow-ons. It's hard to find my own custom ones, I need to scroll through 14 irrelevant presets before hitting mine. 

**Describe the solution you'd like**
Ability to remove the internal presets. Alternatively, just not have any internal presets, let the user add their own custom ones as needed.

**Comment by arndissler** [comment]
Hi Mark,

have you already tried the feature to set a reminder when sending an email This was introduced in mailmindr 1.7.0, and addresses the situation you describe. If this doesn't fit your needs, can you please specify, how the feature should work to fit your needs?

**Comment by Dennisdd** [comment]
Agree, this would be great, including the ability to sort all the presets in the order that you want them to be in. Currently, every new one seems to be added to the end of the list.
